app/forms/recaptcha_enterprise_form.rb

```diff
@@ -24,6 +24,12 @@ def recaptcha_result
           token: recaptcha_token,
           siteKey: IdentityConfig.store.recaptcha_site_key,
           expectedAction: recaptcha_action,
+          userInfo: {
```

Comment:
Account defender is only implemented in Recaptcha enterprise so testing this in lower environments looks tricky.